src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -138,6 +138,11 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
                     // Query parameter with empty value is not supported.
                     unsupportedSearchParameters.Add(query);
                 }
+                else if (string.Equals(query.Item1, "_text", StringComparison.OrdinalIgnoreCase))
```

Comment:
Since `_text` is a well-known common search parameter, I think it would make sense to add it to `KnownQueryParameterNames`

Reply:
Sounds good. Made the change and checked it in.